node/src/neighborhood/mod.rs

```diff
@@ -1245,13 +1392,55 @@ impl Neighborhood {
         return_route_id
     }
 
+    pub fn find_exit_location<'a>(
+        &'a self,
+        source: &'a PublicKey,
+        minimum_hops: usize,
+        payload_size: usize,
```

Comment:
Tenhle argument je zbytecny. Nejspis to vzdy bude 0 nebo neco takoveho. 

Pokud bys to chtel udelat tak, jak by to potesilo Utkarshe. Muzes dokonce vytvorit konstatntu pro tuhle hodnotu s nejakym prihodnym jmenem, aby to nebyla ciste nula, ale mohl bys ten nazev te konstanty udelat nosicem dodatecne informace (s Utkarshem se to vetsinou snazime delat, protoze uz od pradavna se snazime vyhybat komentarum. Psat je pouze v nezbytnych pripadech. Jinak nechat mluvit ten samotny kod. Proto dbame tolik na nazvy, protoze to je to, co ve vysledku asi nejvic rozhoduje, jestli je kod dobry, nebo spatny, ve smyslu srozumitelnosti a nasledne udrzby).